demo/src/SvgDisplay.tsx

```diff
@@ -12,6 +13,7 @@ export default function SvgDisplay(props: SvgDisplayPropTypes) {
     const canvasRef = useRef(null);
     const canvasWidth = 1300;
     const canvasHeight = 1000;
+    const scale = 0.8;
```

Comment:
Don't specify a `scale` (this can be introduced later when implementing zoom functionality)